nessie/jobs/background_job.py

```diff
@@ -38,86 +36,9 @@
-    # return app.config['LOCH_S3_COE_DATA_PATH'] + '/daily/' + today_hash + '-' + today
-    return app.config['LOCH_S3_COE_DATA_PATH']
-
-
-def get_s3_sis_daily_path(cutoff=datetime.now()):
-    today = localize_datetime(cutoff).strftime('%Y-%m-%d')
-    today_hash = hashlib.md5(today.encode('utf-8')).hexdigest()
-    return app.config['LOCH_S3_SIS_DATA_PATH'] + '/daily/' + today_hash + '-' + today
-
-
-def get_s3_sis_api_daily_path(cutoff=datetime.now()):
-    # Path for stashed SIS API data that doesn't need to be queried by Redshift Spectrum.
-    today = localize_datetime(cutoff).strftime('%Y-%m-%d')
-    today_hash = hashlib.md5(today.encode('utf-8')).hexdigest()
-    return app.config['LOCH_S3_SIS_API_DATA_PATH'] + '/daily/' + today_hash + '-' + today
-
-
-def resolve_sql_template_string(template_string, **kwargs):
-    """Our DDL template files are simple enough to use standard Python string formatting."""
-    s3_prefix = 's3://' + app.config['LOCH_S3_BUCKET'] + '/'
-    template_data = {
-        'redshift_schema_asc': app.config['REDSHIFT_SCHEMA_ASC'],
-        'redshift_schema_boac': app.config['REDSHIFT_SCHEMA_BOAC'],
-        'redshift_schema_calnet': app.config['REDSHIFT_SCHEMA_CALNET'],
-        'redshift_schema_canvas': app.config['REDSHIFT_SCHEMA_CANVAS'],
-        'redshift_schema_coe': app.config['REDSHIFT_SCHEMA_COE'],
-        'redshift_schema_coe_external': app.config['REDSHIFT_SCHEMA_COE_EXTERNAL'],
-        'redshift_schema_intermediate': app.config['REDSHIFT_SCHEMA_INTERMEDIATE'],
-        'redshift_schema_metadata': app.config['REDSHIFT_SCHEMA_METADATA'],
-        'redshift_schema_sis': app.config['REDSHIFT_SCHEMA_SIS'],
-        'redshift_schema_student': app.config['REDSHIFT_SCHEMA_STUDENT'],
-        'redshift_iam_role': app.config['REDSHIFT_IAM_ROLE'],
-        'loch_s3_asc_data_path': s3_prefix + get_s3_asc_daily_path(),
-        'loch_s3_calnet_data_path': s3_prefix + get_s3_calnet_daily_path(),
-        'loch_s3_canvas_data_path_today': s3_prefix + get_s3_canvas_daily_path(),
-        'loch_s3_canvas_data_path_historical': s3_prefix + app.config['LOCH_S3_CANVAS_DATA_PATH_HISTORICAL'],
-        'loch_s3_canvas_data_path_current_term': s3_prefix + app.config['LOCH_S3_CANVAS_DATA_PATH_CURRENT_TERM'],
-        'loch_s3_coe_data_path': s3_prefix + get_s3_coe_daily_path(),
-        'loch_s3_sis_data_path': s3_prefix + app.config['LOCH_S3_SIS_DATA_PATH'],
-        'loch_s3_sis_api_data_path': s3_prefix + get_s3_sis_api_daily_path(),
-    }
-    # Kwargs may be passed in to modify default template data.
-    template_data.update(kwargs)
-    return template_string.format(**template_data)
-
-
-def resolve_sql_template(sql_filename, **kwargs):
-    with open(app.config['BASE_DIR'] + f'/nessie/sql_templates/{sql_filename}', encoding='utf-8') as file:
-        template_string = file.read()
-    return resolve_sql_template_string(template_string, **kwargs)
-
-
```

Comment:
All this stuff had to get carted over to `nessie.lib.util` because of circular import problems, but I think it looks nicer in its new home.